wordpress-mobile/WordPress-Android
# App config annotation processing

This PR adds Annotation processing to generate the Remote config defaults for the Firebase remote config. I've created two new modules for annotations and processor. Since this is the first time I've created a module, it would be great if you could let me know if they make sense (naming, location, purpose). 

The code generates the `RemoteConfigDefaults.kt` that is used to define default values of the experiment/feature. This object exposes a map of remote config defaults for the firebase.

To test:
- Add an Experiment/Feature based on the Examples
- Make the project
- Make sure the `RemoteConfigDefaults.kt` is there and contains your new experiment/feature with the default value.

PR submission checklist:
